src/engine_mbedtls.c

```diff
@@ -389,6 +413,11 @@ mbedtls_read(void *engine, const char *ssl_in, size_t ssl_in_len, char *out, siz
         return TLS_OK;
     }
 
+    if (rc == MBEDTLS_ERR_SSL_PEER_CLOSE_NOTIFY) {
+        *out_bytes = total_out;
```

Comment:
Does out_bytes need to be set here? (it was also set on :408)

Reply:
and on 412 fwiw :)

Reply:
cleanedup